numpy/core/fromnumeric.py

```diff
@@ -1814,6 +1814,9 @@ def ravel(a, order='C'):
 
     Notes
     -----
+    It returns a contiguous flattened array.
+    
+    Unlike numpy.reshape, reval supports ``order='K'``.
```

Comment:
I think the additions could be merged into the existing text rather than stand-alone here. The first sentence is a description of the returned value, so perhaps the "Returns" could be modified to read (the 1d is redundant, since the shape specifies the dimensions, but maybe that is OK?)
> y is a contiguous 1d array of the same subtype as `a`, with shape ``(a.size,)``.

The second part could either be part of the sentence below:
> When a view is desired in as many cases as possible, ``arr.reshape(-1)`` may be preferable. However ``ravel`` supports ``K`` in the optional ``order`` argument where ``reshape`` does not.

or part of the See also
> ndarray.reshape: Change the shape of an array, does not allow the ``K`` option for ``order``.

Reply:
Is a 1d array ever non-contiguous?

Reply:
thank you for the review!
I guess there are no non-contiguous 1d arrays. 
However, they can be non-contiguous when we create a view of a 1d array that skips some elements.
https://stackoverflow.com/questions/60848009/can-a-numpy-1d-array-not-be-contiguous